resources/translations/sv.edn

```diff
@@ -33,7 +33,8 @@
                      :invalid-token [:div [:p "Det gick inte att ansluta till ansökningen."] [:p "Vänligen kontrollera koden för inbjudan." [:code "%1"]]]
                      :licenses-not-accepted "Du har inte godkänt användarvillkoren."
                      :organization-does-not-exist "Organisationen \"%1\" hittades inte."
-                     :validation-errors "Var vänlig kontrollera följande:"}
+                     :validation-errors "Var vänlig kontrollera följande:"
+                     :entitlement-end-not-in-future "Utgångsdatumet för rättigheter måste vara i framtiden."}

```

Comment:
We have kept these in alphabetical order. I'm not sure it's an important concern anymore.